test/demo.jl

```diff
@@ -1,6 +1,6 @@
 using ReTest
-using AdvancedHMC, Distributions, ForwardDiff, ComponentArrays, AbstractMCMC
-using LinearAlgebra
+using AdvancedHMC, Distributions, ForwardDiff, Zygote, ComponentArrays, AbstractMCMC
+using LinearAlgebra, ADTypes, LogDensityProblemsAD
```

Comment:
Let's replace LDP with DI. 

```suggestion
using LinearAlgebra, ADTypes, DifferentiationInterface
```